plugins/code-coverage/src/components/FileExplorer/FileExplorer.tsx

```diff
@@ -173,7 +173,9 @@ export const FileExplorer = () => {
   }
   if (!value) {
     return (
-      <Alert severity="warning">No code coverage found for ${entity}</Alert>
+      <Alert severity="warning">
+        No code coverage found for ${String(entity ?? '')}
```

Comment:
I think `humanizeEntityRef` or `stringifyEntityRef` would be best here instead while we're at it

Reply:
also wouldn't this need to be in the changeset?

Reply:
Yep! @RobotSail 👀 https://github.com/backstage/backstage/pull/19512#issuecomment-1688388515

Reply:
🤦  I see that there is already a `stringifyEntityRef` function and we're not just defining it now. Will change that ASAP